### DISCUSSION TITLE
PostgreSQL Driver

### DISCUSSION THREAD

**Discussion by jgubo-mct** [awslabs/aws-mysql-jdbc#32] [Q&A] [answered]
I know, this might not be the perfect place to ask, but does anybody know if there will be a similar project for a RDS Cluster-aware AWS PostgreSQL JDBC Driver?

Many of our applications would extremely benefit from it.

Thanks, Regards Johannes

**Comment by nitesmeh** [comment]
Hi @jgubo-mct  - Thank you for your interest in the MySQL JDBC driver. We continue to evaluate customer interest in all programming languages and databases for future projects.

**Reply by jgubo-mct** [reply]
Thanks Nitesh, please keep us posted here if there are news to announce.

**Reply by hsuamz** [reply]
Hello @jgubo-mct,

Following up with your original question in regards to the PostgreSQL JDBC driver. We've released that driver as a Public Preview and the repository can be found [here](https://github.com/awslabs/aws-postgresql-jdbc).

Please feel free to check out the driver and continue discussion on that repository.

Thanks,

Matthew

**Reply by hsuamz** [reply, marked as answer]
For visibility - Official PostgreSQL support can be found with our JDBC Wrapper project: https://github.com/awslabs/aws-advanced-jdbc-wrapper.
